app/views/citationsNotFound.html

```diff
@@ -0,0 +1,36 @@
+<div class="sub-page-title-block">
+  <h1>No Tickets Found</h1>
+</div>
+<div class="sub-page-container">
+  <p>
+    We could not find any results for the information you provided.
+  </p>
+  <h2>What this means</h2>
+  <p>
+    You may still have pending court cases, or you entered your information incorrectly.
+  </p>
+  <h3>If you just received your ticket</h3>
+  <p>
+    If you just received your ticket, it may not have been entered into the system yet.  Please check back soon.  Tickets have to go through an approval process before being entered into the ticket database.
+  </p>
+  <h3>If you have missed your court date</h3>
+  <p>
+    This website shows only upcoming court dates.  If your court date has passed and you have not resolved your case, there may be a warrant issued for you.  If you have missed the court date for a case involving a warrant and the court date is in the past, it will not show up in our system.  If you know this is your situation, you make look up the court where you missed your date using our <a href="\">court finder</a>.
+  </p>
+  <h3>The court is not participating on this site</h3>
+  <p>
+    This site is for St. Louis County tickets.  Both the unincorporated areas and the municipalities that are part of the county.
+  </p>
+  <p>
+    Currently the following municipalities are supported by this site:
+    <p ng-repeat="municipality in ctrl.supportedMunicipalities">
+      <span ng-if="$first">{{municipality.name}}</span>
+      <span ng-if="!$first">, {{municipality.name}}</span>
```

Comment:
This can be simplified. Always display the name and then in a second `<span>` you can do the `ng-if` with `!$last` that way you don't have to repeat the `municipality.name` part